tasks/zadanie_7.php

```diff
@@ -13,5 +13,10 @@
 $customerB = $params[1]; // tej linijki nie ruszamy :)
 
 // ...
-
-echo "Klient A wydał więcej";
\ No newline at end of file
+if ($customerA > $customerB){
+    echo "Klient A wydał więcej";
+} elseif ($customerA = $customerB) {
```

Comment:
Tutaj powinny być 2x == ;) 
```suggestion
} elseif ($customerA == $customerB) {
```